gapic/src/main/com/google/gapid/views/SettingsDialog.java

```diff
@@ -79,6 +80,10 @@ protected void beforeAnalytics() {
             "Disable replay optimization",
             models.settings.preferences().getDisableReplayOptimization()),
             withSpans(new GridData(SWT.LEFT, SWT.TOP, false, false), 2, 1));
+        enableAllExperimentalFeatures = withLayoutData(createCheckbox(this,
+            "Enable all experimental features (unsupported, required restart)",
```

Comment:
Either "Enable all unsupported experimental features (requires restart)" or drop "unsupported". The way it's written now is a bit confusing. Also notice that it is "requires" not "required".